roles/provision-buildfarm-apb/tasks/main.yml

```diff
@@ -139,94 +145,94 @@
   tags:
   - install-sdk
 
--
-  block:
-    - include: update-secret.yml
-      vars:
-        status_message: Configuring Android SDK container
-        status_percentage: "40"
-        jenkins_uri: "{{ jenkins_route }}"
-        jenkins_preexists: "{{ jenkins_service_preexists }}"
-
-    - name: "Checks if android sdk is already installed"
-      stat:
-        path: "{{ android_sdk_home }}/tools/bin/sdkmanager"
-      register: sdk_folder
-
-    - name: "Install the Android SDK"
-      command: "oc -n {{ namespace }} rsh {{ android_sdk_podname }} androidctl sdk install"
-      when: sdk_folder.stat.exists == False
-
-    -
-      name: Copy default config file
-      copy:
-          src: default.cfg
-          dest: /tmp/sdk.cfg
-          mode: 0777
-      when: ANDROID_SDK_AUTO_CONFIG|default('No') == 'Yes'
+- block:
+  - include: update-secret.yml
+    vars:
+      status_message: Configuring Android SDK container
+      status_percentage: "40"
+      jenkins_uri: "{{ jenkins_route }}"
+      jenkins_preexists: "{{ jenkins_service_preexists }}"
+
+  - name: "Checks if android sdk is already installed"
+    stat:
+      path: "{{ android_sdk_home }}/tools/bin/sdkmanager"
+    register: sdk_folder
+
+  - name: "Install the Android SDK"
+    command: "oc -n {{ namespace }} rsh {{ android_sdk_podname }} androidctl sdk install"
+    when: sdk_folder.stat.exists == False
+
+  - name: Copy default config file
+    copy:
+      src: default.cfg
+      dest: /tmp/sdk.cfg
+      mode: 0777
```

Comment:
I know this is not part of your changes, but is there a reason for it to be this permissive?

Reply:
In tmp dir it does not matter really what mode you use. @odra wdyt?

Reply:
Feel free to change it (doesn't matter as long as you can copy it into the android sdk pod), the file gets deleted together with the temporary provisioning container created by asb